pkg/spiffe/spiffe.go

```diff
@@ -56,6 +60,12 @@ type bundleDoc struct {
 	RefreshHint int    `json:"spiffe_refresh_hint,omitempty"`
 }
 
+type ClusterTrustDomainInfo struct {
+	ClusterID          string
+	TrustDomain        string
+	TrustDomainAliases []string
```

Comment:
Why are there TrustDomainAliases for each ClusterID?
I think TrustDomainAliases is a singleton, and only for "cluster.local", right?

Reply:
Are you suggesting that the aliases should stay all the same for different clusters? so we don't have to store one per cluster? This could be a reasonable constraint, but today: cluster1 and cluster2 have their own meshconfig, and can totally configure different aliases respectively. e.g. cluster1.meshconfig.alias = [td2], cluster2.meshconfig.alias = [td1]
That looks a legitimate config to me as well. WDTY?

Reply:
anyway, we don't need this information for this PR. we can revisit whether above is valid config or not.

removed the aliases from the struct.

Reply:
Sounds good. We don't need to care about the aliases stored in a remote cluster, right?